### ISSUE TITLE
Feature Request: Small "Up Next" beneath Now Playing

### ISSUE THREAD

**Issue by thephenakist** [JasonZigelbaum/jqbx-issues#47]
It'd be nice to know what track is coming up next in some cases, and would help other feature requests (such as crossfading). However, it also spoils the fun of a DJ set where users don't know what to expect. So it should be configurable in the room settings to either be visible or not - but should always be tracked in the background.

**Comment by jtrutwin-zz** [comment]
Can't say I'm a fan of this for anyone other than room mods or admins but if there are rooms that want it +1 as long as it can be configured per room.  :)

**Comment by thephenakist** [comment]
Yeah, that's the idea... It'd be nice if there were a room setting for: "Playlist Style" and you could select "Round Robin" (default) or "Democratic", which would alter the behavior of the playlist.
